specification/index.bs

```diff
@@ -388,6 +388,9 @@ produce an audible performance.
 
 <h5 id="cwmn">Conventional Western music notation (CWMN)</h5>
 
+This notational idiom comprises a set of notational rules common to (but not
```

Comment:
This date range seems too restrictive. I would think the end point should be "the present day" rather than 1900. This is not a historical format; it encompasses what is commonly used and understood today.